src/main/java/seedu/ccacommander/commons/core/LogsCenter.java

```diff
@@ -75,11 +75,11 @@ private static void removeHandlers(Logger logger) {
     }
 
     /**
-     * Creates a logger named 'ab3', containing a {@code ConsoleHandler} and a {@code FileHandler}.
+     * Creates a logger named 'ccacommander9000', containing a {@code ConsoleHandler} and a {@code FileHandler}.
```

Comment:
Should this be ccacommander instead or should the log name be something else?
```suggestion
     * Creates a logger named 'ccacommander', containing a {@code ConsoleHandler} and a {@code FileHandler}.
```

Reply:
yup it should be ccacommander

Reply:
yup it should be ccacommander